rebuild-popular-crates.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
yes i saw :) but the 'x' throws a lot of junk to the console and if you want people to copy paste the output of this script, we need to reduce it.
since this script is not automated but manually invoked, i figured it might be ok.

Reply:
True. No need for -x unless you're debugging. I like the other options because it makes bag feel more like a sensible programming language :-D